subiquity/cmd/server.py

```diff
@@ -73,6 +73,8 @@ def make_server_args_parser():
         help='in dryrun, control basedir of files')
     parser.add_argument(
         '--storage-version', action='store', type=int, default=1)
+    parser.add_argument(
+        '--use-os-prober', action='store', type=bool, default=False)
```

Comment:
I don't think this does what you want, or does it?
`action='store'` indicates that the next argument must be consumed and converted into a boolean, e.g.:

```python
server.py --use-os-prober foo     # bool("foo") => True
server.py --use-os-prober ""      # bool("") => False
server.py --use-os-prober False   # bool("False") => True
server.py --use-os-prober True    # bool("True") => True
server.py --use-os-prober         # -> will fail / missing argument
```

I think this is what you want instead:
```python
parser.add_argument(
    '--use-os-prober', action='store_true')
```

Reply:
Yes, I wanted store_true.